src/main/java/uk/gov/hmcts/divorce/legaladvisor/event/LegalAdvisorMakeServiceDecision.java

```diff
@@ -107,7 +111,9 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
         log.info("Application end state is {}", endState);
 
         serviceApplication.setServiceApplicationDecisionDate(LocalDate.now(clock));
-        if (serviceApplication.getServiceApplicationGranted().toBoolean()) {
+
+        boolean isServiceApplicationGranted = serviceApplication.getServiceApplicationGranted().toBoolean();
```

Comment:
same comment as above about possible null value
also, is the variable needed if it's only used in the if clause?

Reply:
Since we have .mandatory(AlternativeService::getServiceApplicationGranted), it can only ever be Yes or No so will leave the if / else condition as is.